vertx-web-api-service/pom.xml

```diff
@@ -12,6 +12,8 @@
   <artifactId>vertx-web-api-service</artifactId>
 
   <properties>
+    <junit5.version>5.6.0</junit5.version>
```

Comment:
we should consider having this in vertx-dependencies

Reply:
or change the version in vertx-junit5

Reply:
Now the version is aligned to the one in vertx-junit5